resources/cmake/macros/export.cmake

```diff
@@ -24,25 +14,24 @@ endforeach()
 include(CMakePackageConfigHelpers)
 
 write_basic_package_version_file(${CMAKE_CURRENT_BINARY_DIR}/stormConfigVersion.cmake
-     VERSION 0.1.0
-     COMPATIBILITY SameMajorVersion )
+     VERSION ${STORM_VERSION_MAJOR}.${STORM_VERSION_MINOR}.${STORM_VERSION_PATCH}.${STORM_VERSION_TWEAK}
```

Comment:
Do we want to have the Tweak version here? The variable `STORM_VERSION` only has major, minor and patch.

Reply:
Nah, I would not use 4-position versions. We discussed at some point that we would use the tweak version internally to distinguish releases from work-in-progress as we do no longer do that via the git repo.